### DISCUSSION TITLE
How to load my assets(images) using API endpoint? Cant get it to work

### DISCUSSION THREAD

**Discussion by mchammer1041** [GrapesJS/grapesjs#4726] [Q&A]
Right now, I trying to pass assets fetched using my endpoint to config of the AssetManager.

It look like

```
const editor = grapesjs.init({
      storageManager: false,
      assetManager: {assets: ['http://workhorse.local/storage/172/attachments/1639873970743-341105626.jpeg']}

      container : `#test`,

      plugins: [gjsPresetNewsLetter]
    })
```

And when I drag and drop the image block the asset manager does not show my picture

**Comment by Ashley100** [comment]
Shouldn't it be an array of objects with a src property?
Is there any console errors?

**Reply by Ashley100** [reply]
Also make sure that your block has appropriate content type:
 `content: { type: 'image' }`

**Reply by mchammer1041** [reply]
> Shouldn't it be an array of objects with a src property? Is there any console errors?

The documentation says it can be both(array of source links or array of objects). And there are no errors in console related to that.

**Reply by artf** [reply]
I see no issues with using assets as objects or strings in the assetManager, are you able to create a reproducible demo with the issue?

**Reply by mchammer1041** [reply]
> I see no issues with using assets as objects or strings in the assetManager, are you able to create a reproducible demo with the issue?

@artf Thanks for anwering. I cant really. I decided to take a different approach of managing assets because I needed to make a decision.
